Gemfile

```diff
@@ -8,6 +8,10 @@ group :development do
   gem 'rubocop'
 end
 
+group :test do
+  gem 'rspec'
```

Comment:
require the needed rspec dependencies individually

Reply:
Done.